🐛 BUG: Creating a new SvelteKit project fails

### Which Cloudflare product(s) does this pertain to?

Pages

### What version(s) of the tool(s) are you using?

 create-cloudflare version 2.8.3

### What version of Node are you using?

19

### What operating system are you using?

Windows 11

### Describe the Bug

Attempting to scaffold a new SvelteKit project fails due to NPM dependency issues. Just scaffold a new demo app with all the defaults.

### Please provide a link to a minimal reproduction

This can be repeated by using the create-cloudflare CLI application

### Please provide any relevant error logs

```sh
> npm create cloudflare@latest my-svelte-app -- --framework=svelte

// choose all default options

╭ Configuring your application for Cloudflare Step 2 of 3
│
├ Installing dependencies
│ installed via `npm install`
│
├ Adding the Cloudflare Pages adapter
│ npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: my-svelte-app@0.0.1
npm ERR! Found: @sveltejs/kit@1.30.3
npm ERR! node_modules/@sveltejs/kit
npm ERR!   dev @sveltejs/kit@"^1.27.4" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @sveltejs/kit@"^2.0.0" from @sveltejs/adapter-cloudflare@3.0.0
npm ERR! node_modules/@sveltejs/adapter-cloudflare
npm ERR!   dev @sveltejs/adapter-cloudflare@"*" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!

```

--- a/.changeset/wet-geckos-turn.md
+++ b/.changeset/wet-geckos-turn.md
@@ -1,0 +1,5 @@
+---
+"create-cloudflare": patch
+---
+
+C3: Bumped `create-svelte` from `5.3.3` to `6.0.3`

--- a/packages/create-cloudflare/src/frameworks/package.json
+++ b/packages/create-cloudflare/src/frameworks/package.json
@@ -15,7 +15,7 @@
 		"create-react-app": "5.0.1",
 		"create-remix": "2.4.0",
 		"create-solid": "0.3.10",
-		"create-svelte": "5.3.3",
+		"create-svelte": "6.0.3",
 		"create-vue": "3.9.0",
 		"gatsby": "5.12.12",
 		"nuxi": "3.10.0"

